ambry-protocol/src/main/java/com.github.ambry.protocol/MessageInfoListSerde.java

```diff
@@ -43,28 +44,45 @@
   }
 
   int getMessageInfoListSize() {
-    int listcountSize = 4;
     if (messageInfoList == null) {
```

Comment:
I deleted @cgtz's comment by mistake. This was the comment:

minor: would prefer
int size = Integer.Bytes;
if (messageInfoList != null) {
  for ...
}
return size;